Fix flaky  TestElasticsearchIntegrationSmokeTest.testMultiIndexAlias

https://github.com/prestosql/presto/runs/377217851
https://pipelines.actions.githubusercontent.com/sJrKZtZsjYLeyqPixYxcTJp8fX9YNlwyJbmEGSC0oymXVDSpbK/_apis/pipelines/1/runs/1683/signedlogcontent/21?urlExpires=2020-01-07T11%3A18%3A58.4771790Z&urlSigningMethod=HMACV1&urlSignature=YEThFqc8%2FJW8Gx3FQ4HIp%2BISS8KcAxvUeJJszqhFzuA%3D
```
2020-01-07T10:41:14.2472191Z [ERROR] Tests run: 33, Failures: 1, Errors: 0, Skipped: 2, Time elapsed: 70.026 s <<< FAILURE! - in TestSuite
2020-01-07T10:41:14.2479502Z [ERROR] testMultiIndexAlias(io.prestosql.elasticsearch.TestElasticsearchIntegrationSmokeTest)  Time elapsed: 0.107 s  <<< FAILURE!
2020-01-07T10:41:14.2486921Z java.lang.AssertionError: Execution of 'actual' query failed: SELECT count(*) FROM multi_alias
2020-01-07T10:41:14.2490526Z 	at org.testng.Assert.fail(Assert.java:83)
2020-01-07T10:41:14.2494958Z 	at io.prestosql.testing.QueryAssertions.assertQuery(QueryAssertions.java:147)
2020-01-07T10:41:14.2499437Z 	at io.prestosql.testing.QueryAssertions.assertQuery(QueryAssertions.java:103)
2020-01-07T10:41:14.2505029Z 	at io.prestosql.testing.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:153)
2020-01-07T10:41:14.2511645Z 	at io.prestosql.elasticsearch.TestElasticsearchIntegrationSmokeTest.testMultiIndexAlias(TestElasticsearchIntegrationSmokeTest.java:516)
2020-01-07T10:41:14.2523511Z 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-01-07T10:41:14.2527957Z 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-01-07T10:41:14.2533022Z 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-01-07T10:41:14.2536413Z 	at java.lang.reflect.Method.invoke(Method.java:498)
2020-01-07T10:41:14.2541259Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2020-01-07T10:41:14.2549732Z 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
2020-01-07T10:41:14.2554320Z 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
2020-01-07T10:41:14.2559515Z 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
2020-01-07T10:41:14.2566411Z 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
2020-01-07T10:41:14.2568805Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
2020-01-07T10:41:14.2571051Z 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2020-01-07T10:41:14.2573193Z 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2020-01-07T10:41:14.2574658Z 	at java.lang.Thread.run(Thread.java:748)
2020-01-07T10:41:14.2576436Z Caused by: java.lang.RuntimeException: Unknown field multi_alias.nationkey:bigint
2020-01-07T10:41:14.2578662Z 	at io.prestosql.testing.AbstractTestingPrestoClient.execute(AbstractTestingPrestoClient.java:114)
2020-01-07T10:41:14.2580951Z 	at io.prestosql.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:379)
2020-01-07T10:41:14.2583459Z 	at io.prestosql.testing.QueryAssertions.assertQuery(QueryAssertions.java:144)
2020-01-07T10:41:14.2583585Z 	... 16 more
2020-01-07T10:41:14.2586050Z Caused by: java.lang.IllegalArgumentException: Unknown field multi_alias.nationkey:bigint
2020-01-07T10:41:14.2587959Z 	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:216)
2020-01-07T10:41:14.2589981Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:979)
2020-01-07T10:41:14.2592105Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:286)
2020-01-07T10:41:14.2593725Z 	at io.prestosql.sql.tree.Table.accept(Table.java:53)
2020-01-07T10:41:14.2595534Z 	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
2020-01-07T10:41:14.2597750Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:301)
2020-01-07T10:41:14.2600179Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.analyzeFrom(StatementAnalyzer.java:2174)
2020-01-07T10:41:14.2602385Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:1160)
2020-01-07T10:41:14.2604391Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:286)
2020-01-07T10:41:14.2605727Z 	at io.prestosql.sql.tree.QuerySpecification.accept(QuerySpecification.java:144)
2020-01-07T10:41:14.2607200Z 	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
2020-01-07T10:41:14.2608998Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:301)
2020-01-07T10:41:14.2610802Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:311)
2020-01-07T10:41:14.2612638Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:852)
2020-01-07T10:41:14.2614971Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:286)
2020-01-07T10:41:14.2616122Z 	at io.prestosql.sql.tree.Query.accept(Query.java:107)
2020-01-07T10:41:14.2617724Z 	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
2020-01-07T10:41:14.2619809Z 	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:301)
2020-01-07T10:41:14.2621617Z 	at io.prestosql.sql.analyzer.StatementAnalyzer.analyze(StatementAnalyzer.java:278)
2020-01-07T10:41:14.2622966Z 	at io.prestosql.sql.analyzer.Analyzer.analyze(Analyzer.java:83)
2020-01-07T10:41:14.2624619Z 	at io.prestosql.sql.analyzer.Analyzer.analyze(Analyzer.java:75)
2020-01-07T10:41:14.2626220Z 	at io.prestosql.execution.SqlQueryExecution.analyze(SqlQueryExecution.java:219)
2020-01-07T10:41:14.2628006Z 	at io.prestosql.execution.SqlQueryExecution.<init>(SqlQueryExecution.java:178)
2020-01-07T10:41:14.2629843Z 	at io.prestosql.execution.SqlQueryExecution.<init>(SqlQueryExecution.java:95)
2020-01-07T10:41:14.2632244Z 	at io.prestosql.execution.SqlQueryExecution$SqlQueryExecutionFactory.createQueryExecution(SqlQueryExecution.java:712)
2020-01-07T10:41:14.2634412Z 	at io.prestosql.dispatcher.LocalDispatchQueryFactory.lambda$createDispatchQuery$0(LocalDispatchQueryFactory.java:119)
2020-01-07T10:41:14.2636616Z 	at io.prestosql.$gen.Presto_327_99_g55c3966__testversion____20200107_104012_2.call(Unknown Source)
2020-01-07T10:41:14.2639451Z 	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
2020-01-07T10:41:14.2643218Z 	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
2020-01-07T10:41:14.2645348Z 	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
2020-01-07T10:41:14.2646417Z 	... 3 more
2020-01-07T10:41:14.2647557Z 
2020-01-07T10:41:14.6170529Z [INFO] 
2020-01-07T10:41:14.6201605Z [INFO] Results:
2020-01-07T10:41:14.6204141Z [INFO] 
2020-01-07T10:41:14.6204992Z [ERROR] Failures: 
2020-01-07T10:41:14.6206640Z [ERROR]   TestElasticsearchIntegrationSmokeTest.testMultiIndexAlias:516->AbstractTestQueryFramework.assertQuery:153 Execution of 'actual' query failed: SELECT count(*) FROM multi_alias
2020-01-07T10:41:14.6206780Z [INFO] 
2020-01-07T10:41:14.6207459Z [ERROR] Tests run: 33, Failures: 1, Errors: 0, Skipped: 2
```

--- a/presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java
+++ b/presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java
@@ -487,6 +487,12 @@ public class TestElasticsearchIntegrationSmokeTest
                                 .alias("orders_alias")))
                 .actionGet();
 
+        embeddedElasticsearchNode.getClient()
+                .admin()
+                .indices()
+                .refresh(refreshRequest("orders_alias"))
+                .actionGet();
+
         assertQuery(
                 "SELECT count(*) FROM orders_alias",
                 "SELECT count(*) FROM orders");
@@ -511,6 +517,12 @@ public class TestElasticsearchIntegrationSmokeTest
                         .addAliasAction(IndicesAliasesRequest.AliasActions.add()
                                 .index("region")
                                 .alias("multi_alias")))
+                .actionGet();
+
+        embeddedElasticsearchNode.getClient()
+                .admin()
+                .indices()
+                .refresh(refreshRequest("multi_alias"))
                 .actionGet();
 
         assertQuery(

